src/main/kotlin/no/nav/syfo/behandlendeenhet/kafka/BehandlendeEnhetProducer.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.syfo.behandlendeenhet.kafka
+
+import no.nav.syfo.behandlendeenhet.database.domain.PPerson
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class BehandlendeEnhetProducer(
+    private val kafkaProducerBehandlendeEnhet: KafkaProducer<String, KBehandlendeEnhetUpdate>,
+) {
+    fun updateBehandlendeEnhet(
```

Comment:
Ble litt usikker på navnet her. Mulig det er bedre med noe ala `sendBehandlendeEnhetUpdate` for å være litt tydeligere. Så kan man vurdere å bare sende inn personIdent og updatedAt som du foreslår over, og samtidig bruke key i feil-loggingen.

Reply:
Enig, likte dette navnet! 
Likte også at man kan logge key, men da klarer man kanskje ikke å knytte en feilmelding til en person noen gang? Man klarer vel ikke å reversere UUID-utregningen og finne ident?

Reply:
Nei, sant. Da er det kanskje best å sende inn Person (kan utvide objektet med et updatedAt-felt)? 🤔